src/Geocoder/Model/AdminLevel.php

```diff
@@ -11,11 +11,16 @@
 namespace Geocoder\Model;
 
 /**
- * @author William Durand <william.durand1@gmail.com>
```

Comment:
It's not removed, the IDE changed to my tag author during the refactoring. I'll revert it anyway, no problem.